packages/frameworks/react/src/tree-view/stories/tree-view.stories.tsx

```diff
@@ -10,71 +10,27 @@ export default meta
 export const Basic = () => {
   return (
     <TreeView.Root>
-      <TreeView.Label>My Documents</TreeView.Label>
+      <TreeView.Label>Tree</TreeView.Label>
       <TreeView.Tree>
-        <TreeView.Branch id="node_modules" depth={1}>
-          <TreeView.BranchControl id="node_modules" depth={1}>
-            <TreeView.BranchText id="node_modules" depth={1}>
-              📂 node_modules
-            </TreeView.BranchText>
-          </TreeView.BranchControl>
-
-          <TreeView.BranchContent id="node_modules" depth={1}>
-            <TreeView.Item id="node_modules/zag-js" depth={2}>
-              📄 zag-js
-            </TreeView.Item>
-            <TreeView.Item id="node_modules/pandacss" depth={2}>
-              📄 panda
-            </TreeView.Item>
-
-            <TreeView.Branch id="node_modules/@types" depth={2}>
-              <TreeView.BranchControl id="node_modules/@types" depth={2}>
-                <TreeView.BranchText id="node_modules/@types" depth={2}>
-                  📂 @types
-                </TreeView.BranchText>
-              </TreeView.BranchControl>
-
-              <TreeView.BranchContent id="node_modules/@types" depth={2}>
-                <TreeView.Item id="node_modules/@types/react" depth={3}>
-                  📄 react
-                </TreeView.Item>
-                <TreeView.Item id="node_modules/@types/react-dom" depth={3}>
-                  📄 react-dom
-                </TreeView.Item>
-              </TreeView.BranchContent>
-            </TreeView.Branch>
-          </TreeView.BranchContent>
-        </TreeView.Branch>
-
-        <TreeView.Branch id="src" depth={1}>
-          <TreeView.BranchControl id="src" depth={1}>
-            <TreeView.BranchText id="src" depth={1}>
-              📂 src
-            </TreeView.BranchText>
+        <TreeView.Item id="1.0">
+          <TreeView.ItemText>1.0</TreeView.ItemText>
+        </TreeView.Item>
+        <TreeView.Branch id="2.0">
+          <TreeView.BranchControl>
+            <TreeView.BranchTrigger>
+              <TreeView.BranchText>2.0</TreeView.BranchText>
+              <TreeView.BranchIndicator>↕️</TreeView.BranchIndicator>
+            </TreeView.BranchTrigger>
```

Comment:
We could show that it's possible to not use `BranchTrigger` which looks like a basic branch without button. I am not sure if this matters for WAI-ARIA in this case

![image](https://github.com/chakra-ui/ark/assets/6079265/d46e7a74-18ca-4640-a476-2e63b97fdcb3)